README.md

```
@@ -8,6 +8,40 @@ PictoPy is an advanced desktop gallery application that combines the power of Ta
+If you’re new to PictoPy or just starting with Python/Node projects, here’s the simplest way to get things running:
+
+1. **Clone the repository**
+git clone https://github.com/marvan5111/PictoPy.git
+cd PictoPy
+
+2. **Backend setup (Python)**
+- Create a virtual environment:
+  - Windows: `python -m venv venv && venv\Scripts\activate`
+  - Linux/Mac: `python3 -m venv venv && source venv/bin/activate`
+- Install dependencies:
+  ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)